datadog_checks_base/tests/krb5.conf

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Let's put this in a sub-directory

Reply:
I moved it to tests/fixtures/kerberos/krb5.conf
lmk if this is not an appropriate path to store this.